<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AgentFormClass</class>
 <widget class="QMainWindow" name="AgentFormClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>713</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AgentForm</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="labelSA">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QListWidget" name="listWidgetParcels"/>
    </item>
    <item row="1" column="1">
     <widget class="QListWidget" name="listWidgetSearch"/>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="pushButtonDeliver">
      <property name="text">
       <string>Deliver</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="comboBoxParcel"/>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="AgentForm.qrc"/>
 </resources>
 <connections/>
</ui>
